src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -202,39 +202,57 @@ def load_command_table(self, _):
 
     # Offer throughput
     with self.command_group('cosmosdb sql database throughput', cosmosdb_sql_sdk, client_factory=cf_sql_resources) as g:
-        g.show_command('show', 'get_sql_database_throughput')
+        g.command('show', 'get_sql_database_throughput')
```

Comment:
May I know the reason change show_command to command, which will lose the exception handler for 404? Is it by design?

Reply:
I think my branch was not synced. Not sure how this happened. I have reverted it now.